### DISCUSSION TITLE
Restrict adding elements without Row, cell

### DISCUSSION THREAD

**Discussion by SAKTHIVELMARIMUTHU** [GrapesJS/grapesjs#5702] [Q&A] [answered]
Currently element can be drag and dropped in the body it self , I wanna restrict  adding the elements naked insted i want to add these elements only inside the rows and columns

**Comment by mosh-tudor** [comment, marked as answer]
@SAKTHIVELMARIMUTHU Play with `draggable`.

https://grapesjs.com/docs/api/component.html#component

**Reply by SAKTHIVELMARIMUTHU** [reply]
THANK YOU mosh-tudor

**Reply by SAKTHIVELMARIMUTHU** [reply]
@mosh-tudor can you please give me a example for draggable 
For example i want column to dragged in the canvas and every other bloack should be restricted
but inside the column every block can be dropped
